packages/@okta/vuepress-site/docs/reference/okta-expression-language/index.md

```diff
@@ -7,13 +7,13 @@ meta:
 
 # Okta Expression Language overview
 
-This document details the features and syntax of the Okta Expression Language (EL). You can use this language throughout the Okta Admin Console and API for the Okta Classic Engine and Okta Identity Engine.
+This document details the features and syntax of the Okta Expression Language (EL). You can use this language throughout the Admin Console and API for the Okta Classic Engine and Okta Identity Engine.
```

Comment:
I don't see it on Confluence, but the style guide in the devdocs repo still says not to use articles.
```suggestion
This document details the features and syntax of the Okta Expression Language (EL). You can use this language throughout the Admin Console and API for Okta Classic Engine and Okta Identity Engine.
```